cmd/wallet-sdk-gomobile/credentialquery/credentialquery.go

```diff
@@ -31,6 +31,17 @@ type Query struct {
 	goAPICredentialQuery goAPICredentialQuery
```

Comment:
With regards to my comment above - here is where you can use the type from `pkg` directly